ExposedAccessKeys/README.md

```diff
@@ -0,0 +1,144 @@
+# Trusted Advisor Exposed Keys CloudWatch Event Monitor
+
+The Trusted Advisor Exposed Keys CloudWatch Event Monitor demonstrates how to use [AWS Step Functions](https://aws.amazon.com/step-functions/) to orchestrate a serverless workflow using [AWS Lambda](http://aws.amazon.com/lambda/) in response to a [CloudWatch](https://aws.amazon.com/cloudwatch/) Event. On detection of a Trusted Advisor Exposed Access Key CloudWatch Event this workflow deletes the exposed IAM Access Key, summarizes the recent API activity for the exposed key, and sends the summary message to an SNS Topic to notify the subscribers.
+
+This repository contains sample code for all the Lambda functions depicted in the diagram below as well as an AWS CloudFormation template for creating the functions and related resources.
+
+![screenshot for instruction](images/Architecture.png)
+
+### Walkthrough of the Architecture
+1. An IAM Access Key is inadvertently uploaded to one of the popular code repositories monitored by Trusted Advisor (ex: github)
+1. Trusted Advisor detects the key, triggering a Trusted Advisor Exposed Access Keys CloudWatch Event. AWS also temporarily restricts the API calls the key is able to make.
+1. A configured CloudWatch Events rule matches this event, triggering an execution of the `ExposedKey` state machine, which has the following sub-steps:
+    * Delete the exposed IAM Access Key Pair
+    * Summarize recent API activity for the user from CloudTrail
+    * Send security notification message containing summary to `SecurityNotificationTopic` SNS Topic.
+
+## Running the Example
+ 
+#### Option 1: Launch the CloudFormation Template in US East - N. Virginia (us-east-1) 
+The backend infrastructure can be deployed in US East - N. Virginia (us-east-1) using the provided CloudFormation template.
+Click **Launch Stack** to launch the template in the US East - N. Virginia (us-east-1) region in your account:
+
+[![Launch Trusted Advisor Exposed Keys CloudWatch Event Monitor into N. Virginia with CloudFormation](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/images/cloudformation-launch-stack-button.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=ExposedAccessKeysMonitor&templateURL=https://s3.amazonaws.com/exposed-keys-project/exposed_access_keys.output.yaml)
```

Comment:
We'll be making this change as the last change before the pull request is merged as the template and dependencies will likely change. Don't let me forget though because I don't want the 1-click serving an older version!